components/filefeed/event.tsx

```diff
@@ -4,27 +4,47 @@ type Props = {
   event: FileFeedEvent;
 };
 
+const greenEventKeywords = [
+  "added",
+  "completed",
+  "created",
+  "ready",
+  "uploaded",
+];
+const primaryEventKeywords = ["updated", "outcome-acknowledged", "scheduled"];
+const redEventKeywords = ["deleted"];
+
+const eventColors = (topic: string) => {
+  if (greenEventKeywords.some((keyword: string) => topic.includes(keyword))) {
+    return "bg-green-800";
+  } else if (
+    primaryEventKeywords.some((keyword: string) => topic.includes(keyword))
+  ) {
+    return "bg-primary";
+  } else {
+    return "bg-red-800";
+  }
+};
+
 export const Event = ({ event }: Props) => {
+  const capitalizeAndReplace = (topic: string) => {
+    topic = topic.replace("records", "commit");
+    topic = topic.replace("upload:completed", "file created");
+    topic = topic.replace("upload", "file");
```

Comment:
We're probably ok to not map these words and just let them be whatever flatfile sends

Reply:
Removed word replacements